packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -118,9 +118,20 @@ def test_pod_logs(cluster):
 
     with cluster.marathon.deploy_pod_and_cleanup(pod_definition):
         url = get_task_url(cluster, pod_id)
+        url_parts = url.split('/')
+        assert url_parts, 'Invalid URL {}'.format(url)
```

Comment:
the assertion is used to make sure line 130 doesn't fail if url_part is empty
`url_parts[-1]`